preview_generator/preview/builder/image__imconvert.py

```diff
@@ -47,6 +47,10 @@ class ImagePreviewBuilderIMConvert(ImagePreviewBuilder):
         MimetypeMapping("image/x-minolta-mrw", ".mrw"),
     ]
 
+    SUPPORTED_HEIC_MIMETYPE_MAPPING = [
+        MimetypeMapping("image/heic", ".heic"),
```

Comment:
Is it possible to add mimetype mappings for those too (not sure about related extension):
- image/heif > .heif ?, 
- image/heif-sequence > .heif ?
- image/heic-sequence > heic ?

Do you know what is difference between heic and heif ? And if sequenced heif is supported same way as normal one ?

Better support for sequenced heic/heif can be added similarly as video support (multiple images preview), but may need a specific library

Reply:
@inkhey Nice to read https://www.macworld.co.uk/feature/iphone/what-is-heic-3660408/

Reply:
I don't find any `.heif` file to test IM against it

Reply:
But since the IM delegates is called `heic` I don't think we should look for other extension.

Reply:
i just found an .heif here : https://filesamples.com/formats/heif
And many .heic here: https://newsgroup.xnview.com/viewtopic.php?f=60&t=32568#p129428

Difference between heif and heic is that  heic are heif with explicit HEVC compression : 
found here http://fileformats.archiveteam.org/wiki/HEIF and http://nokiatech.github.io/heif/technical.html.

Reply:
I just tested with an .heif file and it works. I'll edit the PR